pyClient/zeth/joinsplit.py

```diff
@@ -515,6 +513,9 @@ def joinsplit(
         outputs_with_a_pk = \
             [(zeth_addr.a_pk, to_zeth_units(value))
              for (zeth_addr, value) in outputs]
+
+        timer = Timer.started()
```

Comment:
Let's drop a comment above this line to explain that this has been added to benchmark the proof generation on the prover - for clarity (although one could argue that a timer dropped in the code is rarely to do something else). Note also that the timer is started before the RPC call -  so there is some latency that should be taken into consideration in the measure.

Reply:
Added a comment.

> Note also that the timer is started before the RPC call - so there is some latency that should be taken into consideration in the measure.

To be clear, this is what I meant by "round-trip time" (hopefully this is negligible compared to the current ~15s proof time).

Reply:
> To be clear, this is what I meant by "round-trip time"

Ah yes, indeed! Thanks for adding a comment